app/views/organisations/toggle_active.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title = "#{action.humanize} #{@organisation.name}" %>
+<% content_for :title, title %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+        href: organisation_path(@organisation),
+      ) %>
+<% end %>
+
+<%= form_for(@organisation, as: :organisation, html: { method: :patch }) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @organisation.name %></span>
+        Are you sure you want to <%= action %> this organisation?
+      </h1>
+      <%= govuk_warning_text text: I18n.t("warnings.organisation.#{action}") %>
```

Comment:
This appears to display slightly differently from the designs (but consistently with https://design-system.service.gov.uk/components/warning-text/), in the the warning icon appears at the top rather than centred.